services/config/src/Provider.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Context } from './Context'
+import { Config } from './types'
+
+const makeContext = (config: Config) => config
```

Comment:
I don't really understand the purpose of this part...

Reply:
Eventually the input might not match what's actually stored in the context, in which case this wouldn't just pass through the argument